src/twisted/__main__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+Make the twisted module executable with the default behaviour of
+running twist.
+"""
+
+from __future__ import division, absolute_import
+
+import sys
+from pkg_resources import load_entry_point
+
+if __name__ == '__main__':
```

Comment:
If I understand `__main__.py`, this will always be true in correct usage and therefore this isn't needed. Removing this would avoid needing to test the empty else branch for 100% coverage.